scalableminds/webknossos
# Render patterns in volume segments to increase distinguishability

- adds patterns to volume segments with varying angles, pattern shapes (stripes vs. grid), frequencies
- changes the color generation code to use a more predictable approach
- uses the jet color scheme for the segments

### URL of deployed dev instance (used for testing):
- https://segmentationpatterns.webknossos.xyz

### Steps to test:
- open a volume tracing or dataset with segmentations and inspect the patterns
- play with the "pattern opacity" setting in the segmentation layer settings
- zoom in and out and ensure that the frequency of the stripes changes to pleasant values when the mag is switched
- check that the color in the "segmentation" tab matches the rendered color

### Issues:
- fixes #4240 

------
